PROPOSALCOPY.MD

```diff
@@ -0,0 +1,42 @@
+## Proposal to use a bot to deposit keys and ether to contract.
+
+As you know, to keep our stacking protocol working we should deposit keys and ether to deposit contract.
+For now, we don't have any automatic solution.
+
+We propose:
+1. Run and fund bot that will deposit buffered ether to contract.
+2. Make refund to the development team that deposited buffered ether to contract before.
+
+## Problem
+
+Ether is placed in buffer, and it must be deposited by calling smart contract. Until now, it was done manually.
```

Comment:
The problem isn't in the buffer, but in that: 1) it requires sc interaction to deposit; 2) protocol apr is earned only on deposited ether — having big buffer hurts apr; 3) the operation is gas-consuming, making it desirable to choose some form of optimal moment / gas price to a) not overspend; b) don't have too much ether in the buffer for too long, hurting the protocol's apr